docs/source/architecture.rst

```diff
@@ -105,7 +105,8 @@ Information Storage
 
 Each node stores several key pieces of information as part of its state:
 
-* List of nodes in the network (also called "connected peers")
+* List of PBFT member nodes in the network (from
+  ``sawtooth.consensus.pbft.peers``)
```

Comment:
Please rebase this PR on master and update this setting to match its new name (`sawtooth.consensus.pbft.members`)

Reply:
Done (updated the setting name here and in glossary.rst).